terraform/modules/aws-brig-email-sending/outputs.tf

```diff
@@ -0,0 +1,14 @@
+# Output required to configure wire-server
+
+output "ses_endpoint" {
+  value = "https://email.${var.region}.amazonaws.com"
```

Comment:
Don't rely on input variables; instead, use the variables obtained from one of the resources that contain a region attribute.

Reply:
No resource in the module exposes this information - as far as I can tell. So instead, I added ad provider directive and set its region with the given variable. and then, I use the `aws_region.current.name` from tiago's code for the interpolation. I tested this, and it works. This way, one can control in which region all resources of the module are going to be created.